### ISSUE TITLE
after updating windows 11 preview build, letters are overlapping in the taskbar ,it will be helpful for lot of ppls  if u fix this asap!

### ISSUE THREAD

**Issue by AswinSid** [zhongyang219/TrafficMonitor#1129]
![image](https://user-images.githubusercontent.com/100089844/161554925-e5714d96-3a4b-4e3e-b414-84854ece24cf.png)

**Comment by amits2249** [comment]
same here

**Comment by cwr31** [comment]
asap? seriously?

**Comment by yogshirsath** [comment]
@zhongyang219  Taskbar widget is overlapping with icons. Tried different Taskbar positions, monitor widget positioning but the same issue.

**Comment by masudr4n4** [comment]
same issue.

**Comment by gulzar5h** [comment]
![image](https://user-images.githubusercontent.com/103045657/161757345-b446f703-a9e7-464c-8c2d-797657fd3cdc.png)

**Comment by seanmamasde** [comment]
Is it because of the new system icon tray spacing or the taskbar icon alignment? I just tried this out but don't know where the problem is.

這個是因為 insider 版本的 Windows 更新工作列系統圖標選單的間距所導致還是說是因為工作列致中所導致? 我最近更新完 Windows 11 下載了這個軟體才看到。

**Comment by YueSe** [comment]
@AswinSid WHO ARE YOU??? Dare to command "fix this asap"!!!

**Comment by GZTimeWalker** [comment]
@AswinSid @seanmamasde @gulzar5h @masudr4n4 @amits2249 @yogshirsath 
plz refer to: #1106 and #777

**Comment by AswinSid** [comment]
> @AswinSid @seanmamasde @gulzar5h @masudr4n4 @amits2249 @yogshirsath 
> plz refer to: #1106 and #777

those are not in english

**Comment by YueSe** [comment]
@AswinSid 
This is not a proper attitude for asking questions or requesting help in the issues. I personally advise the author to ignore this thread. @zhongyang219 
IT IS REALLY ANNOYING TO SEE `**fix this asap**` & `**those are not in english**`.

**Comment by AswinSid** [comment]
> @AswinSid 
> This is not a proper attitude for asking questions or requesting help in the issues. I personally advise the author to ignore this thread. @zhongyang219 
> IT IS REALLY ANNOYING TO SEE `**fix this asap**` & `**those are not in english**`.

sorry bro, i am just new to github and open source stuff.
i am really sorry

**Comment by GZTimeWalker** [comment]
> those are not in english

If all you need is a working binary, there is enough English here to get what you want. If you want to know how it's fixed, take a look at the source code and commit and ask again if you have further questions. thx!

**Comment by wisdmyogeshshirsath** [comment]
> @AswinSid @seanmamasde @gulzar5h @masudr4n4 @amits2249 @yogshirsath plz refer to: #1106 and #777

@GZTimeWalker Temporary files are working properly.

![image](https://user-images.githubusercontent.com/45841790/169878541-61345f33-9221-471d-b970-2bc173818000.png)

@AswinSid @YueSe It is not in English. But modern-day's browser has a feature called Translate to English. 🙂

![image](https://user-images.githubusercontent.com/45841790/169878663-10aa5960-d000-4cdc-b848-f62ef2a4cd23.png)

**Comment by AswinSid** [comment]
> > @AswinSid @seanmamasde @gulzar5h @masudr4n4 @amits2249 @yogshirsath plz refer to: #1106 and #777
> 
> @GZTimeWalker Temporary files are working properly.
> 
> ![image](https://user-images.githubusercontent.com/45841790/169878541-61345f33-9221-471d-b970-2bc173818000.png)
> 
> @AswinSid @YueSe It is not in English. But modern-day's browser has a feature called Translate to English. 🙂
> 
> ![image](https://user-images.githubusercontent.com/45841790/169878663-10aa5960-d000-4cdc-b848-f62ef2a4cd23.png)
> 

did that worked for you??
did the net monitor alligned correctly??

**Comment by wisdmyogeshshirsath** [comment]
> did that worked for you?? did the net monitor alligned correctly??

Yes, it is aligned properly. I have attached a screenshot of my taskbar.

**Comment by RNB0510** [comment]
The temporary fix file links have expired.Can anyone share it with me?I am currently on 22H2
